install_ffmpeg.sh

```diff
@@ -5,12 +5,22 @@ set -ex
 export PATH="$HOME/compiled/bin":$PATH
 export PKG_CONFIG_PATH=$HOME/compiled/lib/pkgconfig
 
+# NVENC only works on Windows/Linux
+if [ $(uname) != "Darwin" ]; then
+  if [ ! -e "$HOME/nv-codec-headers" ]; then
+    git clone https://git.videolan.org/git/ffmpeg/nv-codec-headers.git "$HOME/nv-codec-headers"
+    cd $HOME/nv-codec-headers
+    make -e PREFIX="$HOME/compiled"
+    make install -e PREFIX="$HOME/compiled"
+  fi
+fi
+
 if [ ! -e "$HOME/nasm/nasm" ]; then
   # sudo apt-get -y install asciidoc xmlto # this fails :(
   git clone -b nasm-2.14.02 https://repo.or.cz/nasm.git "$HOME/nasm"
   cd "$HOME/nasm"
   ./autogen.sh
-  ./configure --prefix="$HOME/compiled"
+  ./configure --prefix="$HOME/compiled" --enable-pic --enable-static ${HOST_OS:-} --disable-cli
```

Comment:
Are these options actually valid? Tried those here on OSX and got warnings for each of them (and I think we need to preserve the nasm CLI anyhow since that's how it's invoked; it's a compile-time utility, not a goclient runtime dependency.)

Also when/where is HOST_OS set and what would it be? I don't see it in the env vars on any of my machines.

Reply:
Whoops, updated with the latest version of `install_ffmpeg.sh` from docker-livepeer - HOST_OS is something we need for the MSYS2 build. Should see it now.

Deleted those parameters - looks like I accidentally copied them from the x264 build, where they do exist.